process/current_drive.py

```diff
@@ -226,7 +286,7 @@ def cudriv(self, output: bool):
                     * 1.0e6
                 )
                 faccdfix = auxiliary_cdfix / physics_variables.plascur
-            elif current_drive_variables.iefrffix in [3, 7, 10, 11, 12]:
```

Comment:
Why was `11` removed here? Was it a mistake?

Reply:
The HARE model that we used to have had a switch value of 11. The model was removed but its switch remnants still remained in some places. Was tidying up

Reply:
Ah ok. Could you note in the docs that 11 has been removed?

Reply:
I am updating all of the current drive docs in https://github.com/ukaea/PROCESS/issues/2998. Which captures this change